BitwardenShared/Core/Platform/Services/StateService.swift

```diff
@@ -1713,6 +1765,20 @@ actor DefaultStateService: StateService { // swiftlint:disable:this type_body_le
         showWebIconsSubject.eraseToAnyPublisher()
     }
 
+    func syncToAuthenticatorPublisher() async -> AnyPublisher<(String?, Bool), Never> {
+        activeAccountIdPublisher().flatMap { userId in
+            self.syncToAuthenticatorByUserIdSubject.map { values in
+                let userValue = if let userId {
+                    values[userId] ?? self.appSettingsStore.syncToAuthenticator(userId: userId)
+                } else {
+                    false
+                }
+                return (userId, userValue)
```

Comment:
🎨 Could this be rewritten as the following? IMO it's easier to read and follow the logic.
```suggestion
                guard let userId else {
                    return (nil, false)
                }

                let userValue = values[userId] ?? self.appSettingsStore.syncToAuthenticator(userId: userId)
                return (userId, userValue)
```

Reply:
I like this a lot better. 👍  Adopted in latest commit.